examples/leasing/leasing.cfg

```diff
@@ -0,0 +1,17 @@
+[hil]
+# Config file format (/etc/leasing.cfg):
+# node_list: names of the nodes
+# threshold: number of times that cron job should call the leasing
+# script before releasing the node
+# user_name:
+# password:
+# endpoint: the ip address and the port number
+# which hil is running on
+# status_file: the path to the status file (e.g., /var/lib/leasing)
+
+node_list: 0, 1, 2, 3, 4, 5
```

Comment:
Maybe in the example use non-numeric names (cisco-2, dell-25, etc...). This a bit clearer to me; the numeric names suggest that the field may have deeper semantics.